chain/consensus/mirfbt/node.go

```diff
@@ -0,0 +1,127 @@
+package mirbft
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path"
+	"time"
+
+	"github.com/hyperledger-labs/mirbft"
+	mirCrypto "github.com/hyperledger-labs/mirbft/pkg/crypto"
+	"github.com/hyperledger-labs/mirbft/pkg/grpctransport"
+	"github.com/hyperledger-labs/mirbft/pkg/iss"
+	mirLogging "github.com/hyperledger-labs/mirbft/pkg/logging"
+	"github.com/hyperledger-labs/mirbft/pkg/modules"
+	"github.com/hyperledger-labs/mirbft/pkg/reqstore"
+	"github.com/hyperledger-labs/mirbft/pkg/simplewal"
+	t "github.com/hyperledger-labs/mirbft/pkg/types"
+	"golang.org/x/xerrors"
+)
+
+const (
+	nodeBasePort = 10000
+)
+
+type mir struct {
+	Node  *mirbft.Node
+	OwnID t.NodeID
+	Wal   *simplewal.WAL
+	Net   *grpctransport.GrpcTransport
```

Comment:
Suggestion: Figure out how to make this a general interface where we can hook libp2p instead of making it grpc specific.